docs/sql-guide/functions/function-datetimefromparts.md

```diff
@@ -0,0 +1,55 @@
+---
+title: DATETIMEFROMPARTS()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# DATETIMEFROMPARTS() function
+
+The `DATETIMEFROMPARTS()` function takes integer date and time arguments and returns timestamp.
+
+## Syntax
+
+```
+DATETIMEFROMPARTS(int_year, int_month, int_date, int_hour, int_minute, int_seconds, int_milliseconds)
+```
+
+## Arguments
+
+| Argument         | Data type | Description                                                                                     | Required | Further information |
+|------------------|-----------|-------------------------------------------------------------------------------------------------|---|---|
+| int_year         | integer   | Integer value that specifies year [0,9999]                                                      | Yes | |
+| int_month        | integer   | Integer value that specifies month [1,12]                                                       | Yes | |
```

Comment:
With the descriptions as "integer" is already specified in the data type column, you can leave it out of the description.

And because each of the arguments has a range, we can rename "Further information" to "Values" or "Range" then specify the range of values.

So for this row, the table would look like this:

```
| Argument | Data type | Description | Required | Range |
|---|---|---|---|---|
| month | int | month value | Yes | 1-12 |
```